### ISSUE TITLE
Harmonization with Oxygen - HTML report

### ISSUE THREAD

**Issue by cirulls** [xspec/xspec#92]
The HTML report in the GitHub version of XSpec is different from the HTML report in the XSpec version implemented in Oxygen (see https://github.com/xspec/xspec/pull/79#pullrequestreview-25125234 for a comparison). @cirulls,@AirQuick, and @AlexJitianu agree that the Oxygen version looks better. Therefore the task is to align the HTML report with the Oxygen version. I attach the [related files from Oxygen v14.2](https://github.com/xspec/xspec/files/818789/html-report.zip) @AlexJitianu: are these different in the latest version of Oxygen? If so, can you provide the latest ones?

Note that cd1908a424fd0b2150a5585308c4bb4ecb5bae7a introduced a vertical bar highlighting the link target (see #79 for details). It would be good to integrate this enhancement as it highlights the test when clicking on the link and this is particularly useful when there are lots of tests in the report.

**Comment by AlexJitianu** [comment]
Hi Sandro,
The resources you have attached didn't suffer any change ever since they were added.

**Comment by vincentml** [comment]
I have pushed branch test/pr111 which is related to this issue and pull request #111. This commit e1343b10e6ab7bfe35d655b71cc5c9aee1366155 changes the HTML report to include the CSS inline, and include the success/pending/fail images from the oXygen report also inline. The output HTML report is a single file with no dependencies. To test this, check out branch test/pr111 and run an XSpec test that includes successful, pending, and failing tests. This is not a complete match to the oXygen HTML report but a step in that direction.

**Comment by cirulls** [comment]
For reference, I attach here below screenshots of the HTML report as it is currently generated in `master` after merging #135 and as it is generated with oXygen 19. 

**In this PR**
![image](https://user-images.githubusercontent.com/13256236/34653688-635db448-f3e7-11e7-99b9-cd585af8ef48.png)

**In oXygen 19**
![image](https://user-images.githubusercontent.com/13256236/34653698-8768a32a-f3e7-11e7-9a8c-4c85db9649d3.png)

The work to do is to align the current HTML report with the oXYgen version. I hope it won't be too complicated to adapt oXygen's look and feel - anyone willing to take up this task?

**Comment by galtm** [comment]
Here is a better-looking variation on "**D. Add some kind of alert near the top of the report, if the number of failures isn't 0**":

![image](https://github.com/xspec/xspec/assets/40716346/99cf47cf-4040-4b16-821c-0b7fec86a78d)

In this idea, the message would say "tests" plural if the number of failures is greater than 1, and the alert would not appear at all if the number of failures is 0.

Compared to pink shading alone, this alert message should have better accessibility properties.

**Comment by galtm** [comment]
@xspec/xspec : I'm looking for input from the XSpec community on this issue and #1843. Is anyone still interested in this change in report formatting? (Maybe it is an idea whose time has come and gone...?) If no one thinks it is worthwhile, I'll leave the report as is.

**Comment by AlexJitianu** [comment]
> It seems that older Oxygen versions customized the CSS but newer Oxygen versions take whatever is in this repo without modifying it.

Yes. Oxygen no longer has any patches because as the XSpec project grew and improved they were no longer necessary.

**Comment by galtm** [comment]
Closing this issue because #2055 changes the default style. It has some similarities with the Oxygen style shown in this issue.

To users: If you find some aspect of that Oxygen style that you want the latest XSpec reports to reflect, please open a new issue and indicate what exactly you're looking for (better yet, say why you like it).
